samples/musicfestival-frontend-react/src/pages/SearchPage.tsx

```diff
@@ -8,51 +8,48 @@ import { generateGQLSearchQueryVars } from "../helpers/queryCacheHelper";
 import { getImageUrl, isEditOrPreviewMode } from "../helpers/urlHelper";
 import ReactPaginate from 'react-paginate';
 
-const singleKeyUrl = process.env.REACT_APP_CONTENT_GRAPH_GATEWAY_URL as string
-
 function SearchPage() {
+    const singleKeyUrl = process.env.REACT_APP_CONTENT_GRAPH_GATEWAY_URL as string
+    const ARTIST = "Artist"
+    const OTHERCONTENT = "OtherContent"
+    const options: {value: string; key: string}[] = [
```

Comment:
Can you explain why you need to map this to a dictionary where key and value are the same?